ArcGISRuntimeSDKQt_CppSamples/Layers/PlayAKmlTour/PlayAKmlTour.cpp

```diff
@@ -0,0 +1,177 @@
+// [WriteFile Name=PlayAKmlTour, Category=Layers]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "PlayAKmlTour.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "KmlTour.h"
+#include "KmlTourController.h"
+#include "KmlLayer.h"
+#include "KmlContainer.h"
+#include "KmlNodeListModel.h"
+
+#include <QDir>
```

Comment:
there are some additional includes for this defaultDataPath helper that are needed:

here is an example:
https://github.com/Esri/arcgis-runtime-samples-qt/blob/v.next/ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayKml/DisplayKml.cpp#L29-L34